default/methods/task/task/task_review.py

```diff
@@ -0,0 +1,90 @@
+from methods.regular.regular_api import *
+from shared.database.task.task import Task, TASK_STATUSES
+from shared.database.task.task_event import TaskEvent
+from shared.utils.task.task_update_manager import Task_Update
+from shared.database.discussion.discussion_comment import DiscussionComment
+from shared.utils.task import task_complete
+
+
+@routes.route('/api/v1/task/<int:task_id>/review', methods = ['POST'])
+@Permission_Task.by_task_id(apis_user_list = ["builder_or_trainer"])
+def task_review_api(task_id):
+    """
+
+    """
+    spec_list = [{'comment': {
+        'kind': str,
+        'required': False
+    }},
+        {'action': {
+            'kind': str,
+            'valid_values_list': ['approve', 'request_change']
+        }}
+    ]
+
+    log, input, untrusted_input = regular_input.master(request = request,
+                                                       spec_list = spec_list)
+    if len(log["error"].keys()) >= 1:
+        return jsonify(log = log), 400
+
+    with sessionMaker.session_scope() as session:
+        user = User.get(session)
```

Comment:
We have the `member = get_member(session=session)` option.

I believe with no extra import, already available through regular_api. It provides a central point for changes and reduces code volume. one example https://github.com/diffgram/diffgram/blob/096f7628bd150cc958b084d0347dc501b79c07b8/default/methods/event/event_create.py#L56

I think that pattern is in 3 places on this pull request at least as far as git file search is showing searching `client_id = request.authorization.get('username', None)`

Reply:
Hey Anthony! Thanks for the commets! Sure I think this is a better way. I'll refactor those now.